### PR TITLE
i18n

### PR DESCRIPTION
**Pull request by cap10morgan**
This makes election-notification-works tell notification-works which language to use for notifications based on the user's language-preference.

This has all been krakenstein-tested locally on vote-tomorrow emails.

Part of [this card](https://www.pivotaltracker.com/story/show/117970571).

### CODE REVIEW

File: src/election_notification_works/handlers/util.clj [hunk 1]
```diff
@@ -52,21 +47,6 @@
   [notification-type medium]
   (config [:notification-types notification-type medium :send-at]))
 
-(defn user-district-division
-  "Determines which of the `election`'s district-divisions the `user` is
-  registered to vote in.
-
-  This fn should go away as soon as we make electorate-works remove the
```
Comment (danielglauser) [inline src/election_notification_works/handlers/util.clj]: I'm assuming `electorate-works` now only returns the `district-divisions` that match the voter?

Comment (cap10morgan) [inline src/election_notification_works/handlers/util.clj]: Yes, under a `:district-division` (singular) key.

**Comment by danielglauser** [comment]
LGTM

**Comment by marlabrizel** [comment]
lgtm(too)

**Pull request merged**